Content.Server/Silicons/Laws/SiliconLawEui.cs

```diff
@@ -52,8 +52,10 @@ public override void HandleMessage(EuiMessageBase msg)
             return;
 
         var player = _entityManager.GetEntity(message.Target);
-
         _siliconLawSystem.SetLaws(message.Laws, player);
+
+        if (_entityManager.TryGetComponent<SiliconLawProviderComponent>(player, out var playerProviderComp))
+            _siliconLawSystem.CueEntityMind(player, playerProviderComp?.LawUploadSound);
```

Comment:
```suggestion
            _siliconLawSystem.CueEntityMind(player, playerProviderComp.LawUploadSound);
```
playerProviderComp is never null in the if clause